.travis.yml

```diff
@@ -17,6 +17,12 @@ before_script:
 
 jobs:
   include:
+    - stage: test
```

Comment:
This one should be validation stage right?

Reply:
I thought the stages where only test, build and deploy, that's why I went with this one. If validation is a valid one, it makes sense. I don't like travis ci docs 😛

Reply:
Updated in https://github.com/testing-library/eslint-plugin-testing-library/pull/216/commits/b126b8d4b41fa86a7280305f13331cae9c15b7b2

Reply:
I'd say stages can have any name: https://config.travis-ci.com/ref/stages the only thing is to be consistent around the CI config.